### ISSUE TITLE
HOW TO SET FINETUNE Learning Rate

### ISSUE THREAD

**Issue by LecterF** [hzwer/ECCV2022-RIFE#371]
Hi,
作者您好，我在用您的train.py 代码微调模型，预先load了您训练好的的4.18模型，想请问您后续训练的learning rate 怎么设置

**Comment by hzwer** [comment]
让我拍脑袋想的话，warmup 后维持 1e-5

**Comment by LecterF** [comment]
感谢大佬
